src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_with_native_pubsub.cs

```diff
@@ -0,0 +1,158 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NUnit.Framework;
+
+    public class When_publishing_to_scaled_out_subscribers_with_native_pubsub : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Each_event_should_be_delivered_to_single_instance_of_each_subscriber()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.EndpointsStarted, async (session, c) => { await session.Publish(new MyEvent()); }))
+                .WithEndpoint<SubscriberA_1>()
```

Comment:
would it work to use `.WithEndpoint<SubscriberA>()` twice? the WithEndpoint has an overload which would allow you to configure the instance discriminator.